doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,188 @@
+- VNFM/NFVO, like ONAP, is not used in the process flow for infrastructure verifications, or validations.
+- Upgrades to CNFs, and the respective processes of verifying upgrade procedures and validating (testing) the success and compatibility of upgrades is not in scope.
+
+<a name="1.3"></a>
+## 1.3 Principles and Guidelines
+
+The objectives of the Reference Conformance for cloud infrastrucute is to verify implementations against the reference architecture which satisfies infrastructure needs for CNFs.
+
+The objectives of the Reference Conformance for CNFs is to verify CNF implementations consume resources and behave as expected against the reference architecture.
+
+These core principles will guide RC2 deliverables:
+
+- RC-2 requirements are derived from RM and RA-2 which specify infrastructure capabilities which include compute, memory, storage resource capabilities, performance optimization capabilities, monitoring capabilities.
+
+- Requirements in the RM and RAs that are performance related may not have minimum perfromance criteria identified but where feasible will have tests with metrics to show relevant capabilities are present and working as expected.
+
+- Must/shall conformance criteria are testable as pass/fail and ensure requirement are met to ensure minimum thresholds of functional operability and/or performance behavior. This is the focus of RC-2 since this is what will drive a commercially significant badging program.
+
+- Should/may conformance criteria may be testable or non-testable and provide recommendations or best-practices for functional operability and/or performance behavior. These criteria and associated tests can be very useful for developing, evaluating or deploying a cloud infrastructure but are not critical to a commercially significant badging program.
+
+<a name="1.3.1"></a>
+### 1.3.1 Overarching Standards and Goals
+
+1. Test Harnesses will be portable, or compatible, across all RAs/Distributions which already conform to standard interfaces and services.
+2. Increase probability of CNFs.
+3. Decrease operator onboarding time for both Kubernetes infrastructures and CNFs.
+
+<a name="1.4"></a>
+## 1.4 Best Practices
+
+The following best practices have been adopted to ensure verification and validation procedures are repeatable with consistent quality in test results and RI conformances:
+
+* Standardized test methodology / flow, Test Plan, and Test Case Suites
+* Integration with upstream projects and flow (code, docs, cert criteria, etc.)
+* Leverage Network and Service Models, with identified CNF-specific parameters
+* Standardized conformance criteria
+* Add test cases from operators, which operators already tested in their environment
+
+<a name="1.5"></a>
+## 1.5 Verification methodologies
+
+Perform CNF interoperability verifications against an implementation of CNTT RA2. Upstream projects will define features/capabilities, test scenarios, and test cases to be executed via the OVP Ecosystem.
```

Comment:
are we really testing cnf interoperability -- are we going to test different combination of cnf's?